pan3d/dataset_builder.py

```diff
@@ -225,6 +307,23 @@ def on_set_scale(self, x_scale=None, y_scale=None, z_scale=None, **kwargs):
     # Render Logic
     # -----------------------------------------------------
 
+    def auto_select_coordinates(self):
+        state_update = {}
+        for coordinate in self.state.coordinates:
+            name = coordinate["name"].lower()
+            for axis, accepted_names in coordinate_auto_selection.items():
+                # If accepted name is longer than one letter, look for contains match
+                name_match = [
+                    coordinate["name"]
+                    for accepted in accepted_names
+                    if (len(accepted) == 1 and accepted == name)
+                    or (len(accepted) > 1 and accepted in name)
+                ]
+                if len(name_match) > 0:
+                    state_update[axis] = name_match[0]
+        if len(state_update.keys()) > 0:
```

Comment:
In python I think you can just do `len(state_update)`

Reply:
Changed in [b670a16](https://github.com/Kitware/pan3d/pull/29/commits/b670a16a70e27b9de13458af5130bb75ee8c1f0f)